### PR TITLE
Fix typo in markdown message (+ reword it a bit)

### PR DESCRIPTION
**Pull request by madebr**
Hey @prince-chrismc ,

Great job creating this!

Perhaps we need a (pinned) page on cci with a link to all pages created by bots.
We now have @qchateau's bot, one from @ericlemanissier and now yours

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func main() {
 
 ### :ballot_box_with_check: Selection Criteria:
 
-- No reivews and pull requests was created within 24hrs
+- No reviews or pull requests were created in the last 24hrs
```
Comment (prince-chrismc) [inline main.go:84]: The condition is really `&&`, if there's been reviews (other than approvals) it won't be listed

```suggestion
- No reviews and created in the last 24hrs
```

**Review by prince-chrismc — COMMENTED** [review]
I agree we need a mega thread for all of these bots.

My intention is to just have a list of "easy" reviews for the c3i team and the community reviews that are less active. to speed up merging PRs

File: main.go [hunk 2]
```diff
@@ -81,7 +81,7 @@ func main() {
 
 ### :ballot_box_with_check: Selection Criteria:
 
-- No reivews and pull requests was created within 24hrs
+- No reviews and created in the last 24hrs
```
Comment (madebr) [inline main.go:84]:
```suggestion
- No reviews and pull requests were created in the last 24hrs
```

File: main.go (hunk 1 above)
Comment (madebr) [reply main.go]: No point in arguing with *the author*. :wink:

**Review by prince-chrismc — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (prince-chrismc) [reply main.go]: English is not my first language I appreciate they help =D

**Pull request merged**